src/main/java/eiteam/esteemedinnovation/engineeringtable/BlockEngineeringTable.java

```diff
@@ -38,7 +40,9 @@ public void breakBlock(@Nonnull World world, @Nonnull BlockPos pos, @Nonnull IBl
         TileEntity tileentity = world.getTileEntity(pos);
 
         if (tileentity instanceof TileEntityEngineeringTable) {
-            InventoryHelper.dropInventoryItems(world, pos, (IInventory) tileentity);
+            IItemHandler itemHandler = tileentity.getCapability(ITEM_HANDLER_CAPABILITY, null);
+            assert itemHandler != null;
```

Comment:
You should not use assert except when testing. Change to an if statement, or an exception.